File: samples/cpp/test1/solution/numbers.cpp
#include <iostream>

using namespace std;

int main() 
{
   bool done = false;
   while (!done) 
   {
      cout << "Enter a number, 0 to quit: ";
      int n;
      cin >> n;
      if (n == 0) { done = true; }
      else { cout << "Square: " << n * n << endl; }
   }

   return 0;
}

File: samples/java/hiddenIN1/Numbers.java
//IN HIDDEN 3\n-3\n5\n0
import java.util.Scanner;
public class Numbers
{
   public static void main(String[] args)
   {
      //HIDE
      Scanner in = new Scanner(System.in);
      boolean done = false;
      while (!done)
      {
         //EDIT . . .
         System.out.println("Enter a number, 0 to quit");
         //HIDE
         int n = in.nextInt();
         if (n == 0) done = true;
         else
         {
             int square =  n * n;
             //EDIT . . .
             System.out.println("The square is " + square);
             //HIDE
         }
      }
      //EDIT . . .
   }
}
